tests/dom_tests.js

```diff
@@ -174,6 +174,11 @@ QUnit.test("matches", function(assert) {
   assert.ok(document.querySelector('#baz').matches('.beta'));
 });
 
+QUnit.test("closest", function(assert) {
+  assert.equal(document.querySelector('#bar').closest('#foo'), document.querySelector('#foo'));
+  assert.equal(document.querySelector('#baz').closest('#foo'), document.querySelector('#foo'));
+});
```

Comment:
Sure, will do that! I'm having some problems getting Qunit to work, which version of Qunit and node are you using? It might be beneficial if we could add Qunit as a devDependency and a npm script to run the tests. Would make testing much simpler for new contributors.